<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>409</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>643</width>
    <height>409</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1677721</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>网络流量分类系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>643</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuClassify">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionBegin"/>
    <addaction name="actionBegin_1"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuResults">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>结果</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionShow"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menuClassify"/>
   <addaction name="menuResults"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionBegin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>统计法分类</string>
   </property>
   <property name="toolTip">
    <string>统计法分类</string>
   </property>
   <property name="shortcut">
    <string>Alt+Shift+C</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>分类结果（概率）</string>
   </property>
   <property name="toolTip">
    <string>显示分类结果的混沌矩阵</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionBegin_1">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>正态分布法分类</string>
   </property>
   <property name="toolTip">
    <string>正态分布法分类</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="action_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>分类结果（数值）</string>
   </property>
   <property name="toolTip">
    <string>显示分类到每种类型的结果数据条数</string>
   </property>
   <property name="shortcut">
    <string>Alt+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
